_timeline/2017-12-16-Winter-long-distance-link.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Winter antenna test
+date: 2017-12-16
+---
+We field tested Ubiquiti LiteBeam ac directional antennas for a month in order to determine how a long distance link will behave over a prolonged time in the Canadian climate. [Our Results](https://github.com/tomeshnet/documents/blob/master/technical/20171216_ubiquiti-winter-test.md)
```

Comment:
Ran a [month-long field test]() of Ubiquiti LiteBeam ac directional antennas in order t....